vscode-dotnet-runtime-library/src/Utils/FileUtilities.ts

```diff
@@ -96,8 +96,16 @@ export class FileUtilities extends IFileUtilities
 
     public async read(filePath: string): Promise<string>
     {
-        const output = await fs.promises.readFile(filePath, 'utf8');
-        return output;
+        try
+        {
+            const output = await fs.promises.readFile(filePath, 'utf8');
+            return output;
+        }
+        catch (error: any)
+        {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            throw new Error(`Failed to read file ${filePath}: ${error?.message}`);
```

Comment:
You don't expect filePath to be part of the message? I think this would kill your stack

Reply:
Good point, in that I don't really think this is necessary anymore. Although, the error here is created within the same stack, but avoids including the nodejs internal stack, since it is created within our own code instead of within the node js library, so I actually prefer that, as it often gets truncated 🤔